heaps/heap_sort.py

```diff
@@ -1,8 +1,19 @@
-
+from heaps.min_heap import MinHeap
 
 def heap_sort(list):
     """ This method uses a heap to sort an array.
-        Time Complexity:  ?
-        Space Complexity: ?
+        Time Complexity:  O(log n)
+        Space Complexity: O(n)
     """
```

Comment:
👍  The time complexity is O(n log n) because you add n elements to a heap (n * log n) and then remove n elements from the heap O(n * log n)